modules/tinymce/src/plugins/image/test/ts/browser/DecorativeImageDialogTest.ts

```diff
@@ -42,6 +43,59 @@ UnitTest.asynctest('browser.tinymce.plugins.image.DescriptiveImageDialogTest', (
         sAssertFocused('Save', 'button.tox-button:contains("Save")'),
         sPressEsc,
       ]),
+      Log.chainsAsStep('TBA', 'Image update with empty alt should remove the existing alt attribute', [
```

Comment:
I believe these are duplicate tests, as these look the same as the ones in `modules/tinymce/src/plugins/image/test/ts/browser/A11yImageTest.ts`?

Reply:
It seems so not sure how I missed that maybe the files should be combined.